### ISSUE TITLE
[Feature] Limit order function

### ISSUE THREAD

**Issue by Abhinavcode13** [Clueless-Community/fintech-api#333]
## Are you adding a new feature?
- [x] YES
- [ ] NO

# Are you enhancing the old feature?
 - [] YES
 - [x] NO

 ## Is your feature request related to a problem?
 - [x] YES
 - [ ] NO

## Can you add the feature by yourself?

- [x] YES
- [ ] NO

## Additional Information

Limit Orders: Functions for placing limit orders to buy or sell securities at a specified price or better.

placeLimitOrder(): Places a limit order to buy or sell securities at a specified price or better.

The placeLimitOrder function takes parameters such as symbol (the trading symbol of the security), quantity (the quantity of securities to be bought or sold), limit_price (the specified price at which the order should be executed), and order_type (specifying whether it is a 'buy' or 'sell' order).
The function connects to the API or trading platform using the connect_to_api function (which needs to be implemented based on the specific API documentation).
It constructs the limit order request using the provided parameters.
The api.place_order method is called to place the order, passing the order request as a parameter.
The response is checked to determine if the order was successfully placed. If successful, it retrieves the order ID and prints a success message. Otherwise, it prints an error message.
Finally, the function disconnects from the API or trading platform using the disconnect_from_api function (which needs to be implemented based on the specific API documentation).

**Comment by Abhinavcode13** [comment]
Please assign me this issue under gssoc 23 @ighoshsubho 
thanks

**Comment by ighoshsubho** [comment]
Sure go ahead @Abhinavcode13 ;)

**Comment by ighoshsubho** [comment]
Hello @Abhinavcode13, this is Version 2 of Fintech API repository, there had been few major updates, please go through the docs to make further contributions. Thanks.

**Comment by bhavya1857** [comment]
@ighoshsubho so can i still contribute for this issue

**Comment by bhavya1857** [comment]
why isn't this issue closed

**Comment by ighoshsubho** [comment]
> why isn't this issue closed

Can you work on it @bhavya1857?

**Comment by bhavya1857** [comment]
yes, but why isn't this issue closed? And what do you mean by version 2

**Comment by ighoshsubho** [comment]
It was raised by @Abhinavcode13, and it was not done as expected. I kept it open so that he or any other contributor if comes can work on making this feature.

Version 2 is for adding more features to this API repo. Now the entire repo is:

1. Modular
2. Has request validation
3. Auth for certain routes
4. API docs

These were not there in prev version.

Any more clarification @bhavya1857?

**Comment by bhavya1857** [comment]
Oh ok, I can do this

**Comment by ighoshsubho** [comment]
Sure thing!

**Comment by bhavya1857** [comment]
is this the updated version
![Uploading github1.png…]()
?

**Comment by ighoshsubho** [comment]
The pic has not loaded and its pointing to this page URL.

Anyways the repo itself is updated. Fork the repo and then clone it locally. Else sync the previous changes. and your cloned repo will get upgraded to v2.

Then work on this feature locally.

**Comment by bhavya1857** [comment]
i'm sorry i couldn't figure this out, please assign it to someone else
